crates/dht/src/time.rs

```diff
@@ -811,26 +827,34 @@ mod tests {
         let store = Arc::new(Kitsune2MemoryOpStore::default());
         store
             .process_incoming_ops(vec![
-                MetaOp {
-                    op_id: OpId::from(bytes::Bytes::from(vec![7; 32])),
-                    timestamp: origin_timestamp,
-                    op_data: vec![],
-                },
-                MetaOp {
-                    op_id: OpId::from(bytes::Bytes::from(vec![23; 32])),
-                    timestamp: origin_timestamp,
-                    op_data: vec![],
-                },
-                MetaOp {
-                    op_id: OpId::from(bytes::Bytes::from(vec![7; 32])),
-                    timestamp: (now - UNIT_TIME).unwrap(),
-                    op_data: vec![],
-                },
-                MetaOp {
-                    op_id: OpId::from(bytes::Bytes::from(vec![29; 32])),
-                    timestamp: (now - UNIT_TIME).unwrap(),
-                    op_data: vec![],
-                },
+                Kitsune2MemoryOp::new(
+                    OpId::from(bytes::Bytes::from(vec![7; 32])),
+                    origin_timestamp,
+                    vec![],
+                )
+                .try_into()
+                .unwrap(),
+                Kitsune2MemoryOp::new(
+                    OpId::from(bytes::Bytes::from(vec![23; 32])),
+                    origin_timestamp,
+                    vec![],
+                )
+                .try_into()
+                .unwrap(),
+                Kitsune2MemoryOp::new(
+                    OpId::from(bytes::Bytes::from(vec![7; 32])),
```

Comment:
Is this one okay to still be duplicated?

Reply:
Apparently yes, the test passe whether I change that or not. That suggests I need less test data in there :)